fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/mod.rs

```diff
@@ -1604,6 +1946,18 @@ mod tests {
             self
         }
 
+        fn command_one_pack_on(mut self, pack_id: usize) -> Self {
+            self.write_by_name(
+                format!("OVHD_COND_PACK_{}_PB_IS_ON", 1 + (pack_id == 2) as usize).as_str(),
+                true,
+            );
+            self.write_by_name(
+                format!("OVHD_COND_PACK_{}_PB_IS_ON", 1 + (pack_id == 1) as usize).as_str(),
+                false,
+            );
+            self
+        }
```

Comment:
Feels to me it's to complicated for the purpose, but I might miss something !?  What are you trying to do exactly?

Reply:
Yes I think you are right... The idea is that the `pack_id` is left on, and the opposite pack is switched off. I've changed it now to make it simpler and easier to read.